src/ape/_cli.py

```diff
@@ -104,5 +104,5 @@ def get_command(self, ctx, name):
     callback=display_config,
     help="Show configuration options (using `ape-config.yaml`)",
 )
-def cli():
-    pass
+def cli(helper):
```

Comment:
Or maybe just `ape_cli`?

idk not great with names today

Reply:
Here are some ideas:

* `plugin_helper`
* `cli_helper`
* `cli_tool`
* `utilities`
* `ape`
* `plugin_tools`
* `plugin`
* `sdk`
* `ape_sdk`
* `ape_cli_sdk`

 So far I like `utilities`, `helper`, and `ape_cli_sdk` the best.

Reply:
We often just call this `context` in click apps.

Reply:
Maybe `ape_cli_tools` is the most descriptive

Reply:
`ape` would be totally confusing lmao

`ape_cli` would not be too bad (`ape_cli.logger`, `ape_cli.abort(...)`, etc.) It would be unlikely to be re-used in that way (e.g. `from ape import cli as ape_cli` wouldn't be as nice as `from ape.cli import verbose_option`)

Don't want to be too verbose here, as there will likely be many other arguments to such CLI functions. Food for thought.

Reply:
`ape_cli_helper` is more like what it is though (or `ape_cli_context`). `ape_cli` makes me think it is an instance of a `click.Group` or something.

Reply:
Definitely better